jboss/container/wildfly/launch/keycloak/added/keycloak.sh

```diff
@@ -668,8 +668,10 @@ function configure_subsystem() {
     # SSO Server 7.0
     realm_certificate=`$CURL -H "Accept: application/json" -H "Authorization: Bearer ${token}" ${sso_service}/admin/realms/${SSO_REALM} | grep -Po '(?<="certificate":")[^"]*'`
     if [ -z "$realm_certificate" ]; then
-      #SSO Server 7.1
-      realm_certificate=`$CURL -H "Accept: application/json" -H "Authorization: Bearer ${token}" ${sso_service}/admin/realms/${SSO_REALM}/keys | grep -Po '(?<="certificate":")[^"]*'`
+      #SSO Server 7.1 and newer. If is 7.5 we skip certificate with use=ENC
+      realm_certificate=`$CURL -H "Accept: application/json" -H "Authorization: Bearer ${token}" ${sso_service}/admin/realms/${SSO_REALM}/keys | $(jq '.keys[] | select( .certificate != null and .use == "SIG")') | $(jq '.certificate') `
```

Comment:
This change introduce a new dependency on jq command line. 
Is this fix: https://github.com/jboss-container-images/redhat-sso-7-openshift-image/pull/192/commits/214fca4461236a6430bf21c1f2e53fbcb53fec81
possibly a better candidate (no use of jq)? It covers 7.0, 7.1 to 7.4 and 7.5.

Reply:
@jfdenise fix applied on the script